bundles/best_practices.rst

```diff
@@ -463,6 +463,17 @@ API is being used. The following code, would work for *all* users::
         }
     }
 
+Resources
+---------
+
+If the bundle references any resources (config files, templates, translation
```

Comment:
For Twig templates the namespaces would be `@App` instead of `@AppBundle`. And furthermore, you would leave out the `Resources/views` portion of the template path (because Twig namespaces are not handled by the kernel resource locater but are a core feature of Twig itself).